pom.xml

```diff
@@ -110,9 +110,10 @@
     <spark2.version>2.4.4</spark2.version>
     <spark3.version>3.0.0</spark3.version>
     <avro.version>1.8.2</avro.version>
-    <scala.version>2.11.12</scala.version>
+    <scala-11.version>2.11.12</scala-11.version>
```

Comment:
Hi, why do we use `scala-11` to replace `scala11`?

Reply:
They are the same, just feel like ```scala-11``` is more readable. Like hbase it follows this format: https://github.com/apache/hbase/blob/master/pom.xml#L1599.

Reply:
Ok, so wdyt about `spark3` or `spark-3`? IMO, it would be better to keep consistency in one project.

Reply:
Make sense to me. For the consistency reason, I converted ```scala-11``` back to ```scala11```.